packaging/config/agent.yaml

```diff
@@ -93,5 +93,7 @@
 ###############################################################################
 
 ## Facts gathering service URL
+## URL to an AMQP service handling communication between the Trento Agent and Server components
+## For more details on the URI scheme, please refer to https://www.rabbitmq.com/docs/uri-spec
 
-# facts-service-url: amqp://guest:guest@localhost:5672
\ No newline at end of file
+# facts-service-url: amqp://guest:guest@localhost:5672
```

Comment:
Also here:
`facts-service-url: amqp://guest:guest@localhost:5672` -->  `facts-service-url: amqp://guest:guest_password@localhost:5672/vhost`

Reply:
See example [from official docs](https://www.rabbitmq.com/docs/uri-spec#appendix-a-examples):
```
amqp://user:pass@host:10000/vhost
```

Reply:
my rationale with leaving as it is that in the default configuration we need to put the default values, as in:
https://github.com/trento-project/agent/blob/58fd08c17c2152137c5eb6d7f602abc8a836ce06/cmd/start.go#L124

and I don't really want to change the current default. I think the explanation and the reference should be enough to fully understand the URL scheme, isn't it?

Reply:
Oh, I see :)
Yeah, sure :)